frontend/src/AssignmentView/AssignmentView.tsx

```diff
@@ -2,37 +2,59 @@ import React from "react";
 import Button from "react-bootstrap/Button";
 import { Form } from "react-bootstrap";
 import Container from "react-bootstrap/Container";
-import ProgressBar from "react-bootstrap/ProgressBar";
 import StudentAssignmentView from "./StudentAssignmentView";
 import ProfessorAssignmentView from "./ProfessorAssignmentView";
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import { useCookies } from "react-cookie";
+
+const AssignmentView = () => {
+  const mode: "student" | "faculty" = "faculty";
+  const [assignmentName, setAssignmentName] = useState("Loading Assignment..."); // dummy value before testing with endpoint
+  const [cookies, setCookies] = useCookies(["jwt"]);
+  const params = useParams();
+
+  fetch(
+    "http://localhost:8080/class/1/1", //"http://localhost:8080/class/${params.classId}/${params.assignmentId}", will change
```

Comment:
Since @1strangequark's PR is out I would probably just change it to the comment and then quickly merge and approve @1strangequark 's PR before yours